app/helpers/application_form_helper.rb

```diff
@@ -108,4 +108,14 @@ def application_form_summary_rows(
       { key: { text: key }, value: { text: value }, actions: actions || [] }
     end
   end
+
+  def display_work_history_banner?(application_form)
```

Comment:
```suggestion
  def application_form_display_work_history_before_teaching_qualification_banner?(application_form)
```

The helpers are global methods across all the views, so for a convention we've prefixed them with the model they're related to `application_form`.